enterprise/server/cmd/ci_runner/main.go

```diff
@@ -115,6 +116,14 @@ func main() {
 
 	ctx := context.Background()
 
+	// Bazel needs a HOME dir; ensure that one is set.
+	if err := ensureHomeDir(); err != nil {
+		fatal(status.WrapError(err, "ensure HOME"))
+	}
+	// Make sure PATH is set.
+	if err := ensurePath(); err != nil {
+		fatal(status.WrapError(err, "ensure PATH"))
+	}
```

Comment:
This change was needed because `go-git` falls back to the `git` binary to handle `file://` URLs. This wasn't a problem for the CI runner test because that test is invoking the CI runner binary directly, and inheriting PATH from the host process.